app/assets/json/translations/react/en.json

```diff
@@ -64,10 +64,12 @@
   "listings.reservedCommunityType.habitat": "Habitat Greater San Francisco",
   "listings.reservedCommunityType.senior": "Senior Building",
   "listings.reservedCommunityType.veteran": "Veterans Building",
-  "listings.unitTypes.1 BR": "1 BR",
-  "listings.unitTypes.2 BR": "2 BR",
-  "listings.unitTypes.3 BR": "3 BR",
-  "listings.unitTypes.SRO": "SRO",
+  "listings.unitTypes.1 BR": "1 Bedroom",
+  "listings.unitTypes.2 BR": "2 Bedroom",
+  "listings.unitTypes.3 BR": "3 Bedroom",
+  "listings.unitTypes.4 BR": "4 Bedroom",
+  "listings.unitTypes.5 BR": "5 Bedroom",
+  "listings.unitTypes.SRO": "SRO (single-room occupancy)",
```

Comment:
SRO should just be SRO for now. We don't have room for the explanation in the parentheses in the tables we have